framework/include/task_manager/task_manager.h

```diff
@@ -240,10 +246,12 @@ int task_manager_restart(int handle, int timeout);
  *			TM_NO_RESPONSE : Ignore the response of request from task manager\n
  *			TM_RESPONSE_WAIT_INF : Blocked until get the response from task manager\n
  *			integer value : Specifies an upper limit on the time for which will block in milliseconds
+ * 		If reply is not NULL(=SYNC), timeout should not be set to TM_NO_RESPONSE.
+ * @param[out] reply the reply msg for sync. If this is NULL, task_manager_unicast works asynchronously.
  * @return On success, OK is returned. On failure, defined negative value is returned.
  * @since TizenRT v2.0 PRE
  */
-int task_manager_unicast(int handle, void *msg, int msg_size, int timeout);
+int task_manager_unicast(int handle, tm_unicast_msg_t *send_msg, tm_unicast_msg_t *reply_msg, int timeout);
```

Comment:
As API is changed, corresponding information also should be changed

Reply:
done